lib/components/admin/print-field-trip-layout.js

```diff
@@ -0,0 +1,235 @@
+import PrintableItinerary from '@opentripplanner/printable-itinerary'
+import React, { Component } from 'react'
+import { Button } from 'react-bootstrap'
+import { connect } from 'react-redux'
+import styled from 'styled-components'
+
+import * as callTakerActions from '../../actions/call-taker'
+import * as fieldTripActions from '../../actions/field-trip'
+import TripDetails from '../narrative/connected-trip-details'
+import { getTripFromRequest, lzwDecode } from '../../util/call-taker'
+import { ComponentContext } from '../../util/contexts'
+import { getTitle } from '../../util/state'
+
+// Styles specific for rendering PrintFieldTripLayout.
+const PrintLayout = styled.div`
```

Comment:
Let's put all of these styles in a `styled.js` file (which is the approach we take elsewhere in this repo).

Reply:
A separate `styled-print.js?` or a variable in `styled.js` for the print styles?

Reply:
I like the `styled-print.js` idea.

Reply:
updated in https://github.com/opentripplanner/otp-react-redux/pull/398/commits/a266e27e162309aa649d2d1575b0781773dc66af.